Add window support and disable interval trigger

Aenos85 opened the pull request:
- Update trigger syntax to new one https://github.com/TheRealSimon42/Eco-Heating-Ultimate/issues/2

- Add a option to disable the timed automation run (actual all 5 minutes). Now you can disable this trigger completely. That is necessary in my case because when I adjust the temperature in the dashboard directly at the climate the automation overrides the temperature all 5 minutes back to standard. Additionally, you can change the interval from /5 to any interval you want.

- Added Window settings for state opened. In the old behavior the automation do not recognize when the window state is changed. Only after the 5-minute interval and then it do nothing. Now you can configure the window open state. -> You can enable the window change state trigger with a wait time option for state open -> You can now choose which action you want to be executed when window is opened. You can select between: => "None" (same behaviour like before) => "off/5 degrees" (sets the temperature to 5 degrees) => "Away temperature" => "Eco temperature" (related to the daytime) => "Comfort temperature" (related to the daytime)

RomanTBA committed:
Changed the following topics:
- Update trigger syntax to new one
  https://github.com/TheRealSimon42/Eco-Heating-Ultimate/issues/2

- Add a option to disable the timed automation run (actual all 5 minutes). Now you can disable this trigger completely. That is necessary in my case because when I adjust the temperature in the dashboard directly at the climate the automation overrides the temperature all 5 minutes back to standard. Additionally, you can change the interval from /5 to any interval you want.

- Added Window settings for state opened.
  In the old behavior the automation do not recognize when the window state is changed. Only after the 5-minute interval and then it do nothing.
  Now you can configure the window open state.
  -> You can enable the window change state trigger with a wait time option for state open
  -> You can now choose which action you want to be executed when window is opened. You can select between:
     => "None" (same behaviour like before)
     => "off/5 degrees" (sets the temperature to 5 degrees)
     => "Away temperature"
     => "Eco temperature" (related to the daytime)
     => "Comfort temperature" (related to the daytime)

TheRealSimon42 commented:
Amazing, thank you! Just a quick Question, does the Blueprint still work when there is no Window-Contact added? That was the whole reason going for the Interval, but since it was my 1st blueprint, maybe I was just too stupid for finding a way around it 😅

RomanTBA committed:
Adjust the window entity default for a possible missing/empty entity

Aenos85 commented:
I tested it and it do not worked without an window entity. But I adjust the default of the window entity to set an empty object as default. In this way the trigger is fine without an entity, especially because you can disable the window trigger.

`default: {}`

RomanTBA committed:
Change description typo

Aenos85 committed:
Change description typo

Aenos85 committed:
Revert "Change description typo"

This reverts commit 4626684e8f23a7b474001d71adc552fa9e025256.

Aenos85 committed:
Revert "Adjust the window entity default for a possible missing/empty entity"

This reverts commit 6de5f698f8d5a6e6ebc6d30c8ff87d9e619a7773.

Aenos85 committed:
Revert "Changed the following topics:"

This reverts commit d5afbf1548bc66c90c0fbd66ae0e5b029741319c.

Aenos85 committed:
Changed the following topics:
- Update trigger syntax to new one
  https://github.com/TheRealSimon42/Eco-Heating-Ultimate/issues/2

- Add a option to disable the timed automation run (actual all 5 minutes). Now you can disable this trigger completely. That is necessary in my case because when I adjust the temperature in the dashboard directly at the climate the automation overrides the temperature all 5 minutes back to standard. Additionally, you can change the interval from /5 to any interval you want.

- Added Window settings for state opened.
  In the old behavior the automation do not recognize when the window state is changed. Only after the 5-minute interval and then it do nothing.
  Now you can configure the window open state.
  -> You can enable the window change state trigger with a wait time option for state open
  -> You can now choose which action you want to be executed when window is opened. You can select between:
     => "None" (same behaviour like before)
     => "off/5 degrees" (sets the temperature to 5 degrees)
     => "Away temperature"
     => "Eco temperature" (related to the daytime)
     => "Comfort temperature" (related to the daytime)
